src/StateService/Settings.cs

```diff
@@ -6,13 +6,15 @@ internal class Settings
     {
         public string Path { get; }
         public bool FullState { get; }
+        public bool AutoStart { get; }
 
         public static Settings Default { get; private set; }
 
         private Settings(IConfigurationSection section)
         {
             Path = string.Format(section.GetValue("Path", "Data_MPT_{0}"), ProtocolSettings.Default.Magic.ToString("X8"));
             FullState = section.GetValue("FullState", false);
+            AutoStart = section.GetValue("AutoStart", false);
```

Comment:
How about `AutoVerify`?
Because `StateService` has two features: 1. calculate mpt root 2. be state verifier.
Since calcaluting mpt root starts as long as StateService installed, this setting is to start verification. So `AutoStart` will be confusing. How do you think?

Reply:
Done